test/e2e/network/network_tiers.go

```diff
@@ -136,14 +136,10 @@ var _ = SIGDescribe("Services [Feature:GCEAlphaFeature][Slow]", func() {
 
 func waitAndVerifyLBWithTier(jig *framework.ServiceTestJig, ns, svcName, existingIP string, waitTimeout, checkTimeout time.Duration) string {
 	var svc *v1.Service
-	if existingIP == "" {
-		// Creating the LB for the first time; wait for any ingress IP to show
-		// up.
-		svc = jig.WaitForNewIngressIPOrFail(ns, svcName, existingIP, waitTimeout)
-	} else {
-		// Re-creating the LB; wait for the ingress IP to change.
-		svc = jig.WaitForNewIngressIPOrFail(ns, svcName, existingIP, waitTimeout)
-	}
+
+	// Creating the LB for the first time; wait for any ingress IP to show
+	// up. Or re-creating the LB; wait for the ingress IP to change.
+	svc = jig.WaitForNewIngressIPOrFail(ns, svcName, existingIP, waitTimeout)
```

Comment:
i _think_ this is correct. ref: https://github.com/kubernetes/kubernetes/blob/9e6ed2713e3a33040a47e721ebf9f5bd2ec1fa10/test/e2e/framework/service_util.go#L535-L548

Reply:
@newlit123 Do you agree with the original or modified?

Reply:
agree.